src/unix/aix.c

```diff
@@ -881,24 +886,94 @@ void uv__fs_event_close(uv_fs_event_t* handle) {
 
 
 char** uv_setup_args(int argc, char** argv) {
-  return argv;
+  char** new_argv;
+  size_t size;
+  char* s;
+  int i;
```

Comment:
Yes, as @bnoordhuis noticed I copied the bulk for this from process_title.c which has exactly the same set of declarations in its uv_setup_args.